plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java

```diff
@@ -9032,6 +9033,62 @@ public void testAutoPurgeProperty()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testExternalTablePurgeProperty() throws IOException {
+
+        TrinoFileSystem fileSystem = getTrinoFileSystem();
+        Location tempDir = Location.of("local:///temp_" + UUID.randomUUID());
+        fileSystem.createDirectory(tempDir);
+        Location dataFile = tempDir.appendPath("text.text");
+        try (OutputStream out = fileSystem.newOutputFile(dataFile).create()) {
+            out.write("helloXworld\nbyeXworld".getBytes(UTF_8));
+        }
+        // Table properties
+        String tableName = "test_external_purge_table";
+        String schema =getSession().getSchema().get();
+        StringJoiner propertiesSql = new StringJoiner(",\n   ");
+        propertiesSql.add(format("external_location = '%s'", tempDir));
+        propertiesSql.add("format = 'TEXTFILE'");
+
+        @Language("SQL") String createTableSql = format("" +
+                        "CREATE TABLE %s.%s.%s (col1 varchar,col2 varchar)\n" +
+                        "WITH (\n" +
+                        "   %s\n" +
+                        ")",
+                catalog,
+                schema,
+                tableName,
+                propertiesSql);
+        assertUpdate(createTableSql);
+
+        TableMetadata tableMetadataDefaults = getTableMetadata(catalog, schema, tableName);
+        assertThat(tableMetadataDefaults.metadata().getProperties()).doesNotContainKey(EXTERNAL_TABLE_PURGE);
+
+        assertUpdate(format("DROP TABLE %s", tableName));
```

Comment:
No need to use `format` method in this case. https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#prefer-string-formatting